### ISSUE TITLE
Add paper evidence with DATABASE paper reference for WB descriptions

### ISSUE THREAD

**Issue by valearna** [valearna/genedesc_generator#63]
Add Paper evidence to all rows in the generated ace files

**Comment by valearna** [comment]
I can't find a WBPaperID for the database paper on Alliance gene descriptions. Do we need to add it to WB @rankishore ?

**Comment by rankishore** [comment]
You are right, I couldn't find it in WB either, so I just created it. @valearna it has been created in WB as WBPaper00065943. Please see http://tazendra.caltech.edu/~postgres/cgi-bin/paper_editor.cgicurator_id=two324&action=Search&data_number=00065943

**Comment by valearna** [comment]
That's great, thanks @rankishore !
